src/components/ExtractingFormOcrControl.vue

```diff
@@ -30,7 +31,10 @@ export default {
       return 'default'
     },
     isOcrLanguage() {
-      return !this.isoLang || !!find(this.ocrLanguages, (language) => language.iso6392 === this.isoLang)
+      return (
+        (this.hasTesseract && !this.isoLang) ||
+        !!find(this.ocrLanguages, (language) => language.iso6392 === this.isoLang)
```

Comment:
`!!find(this.ocrLanguages, (language) => language.iso6392 === this.isoLang` should be in a computed property